diff --git a/Oxygen/sonic3air/build/_cmake/CMakeLists.txt b/Oxygen/sonic3air/build/_cmake/CMakeLists.txt
index f7b0cc8..0fbdc4d 100644
--- a/Oxygen/sonic3air/build/_cmake/CMakeLists.txt
+++ b/Oxygen/sonic3air/build/_cmake/CMakeLists.txt
@@ -32,7 +32,7 @@ if (CMAKE_CXX_COMPILER_ID STREQUAL "GNU")
 	#set(CMAKE_CXX_FLAGS "-Wall")
 	set(CMAKE_CXX_FLAGS_DEBUG "-g2")		# Include debug information
 	set(CMAKE_CXX_FLAGS_RELEASE "-g0")		# Leave out debug information
-	set(CMAKE_CXX_FLAGS_RELEASE "-O3")		# Full optimization
+	set(CMAKE_CXX_FLAGS_RELEASE "-O0")		# Full optimization
 	add_compile_options(-Wno-psabi)
 
 	# Needed under Linux to avoid link error from minizip: "relocation r_x86_64_pc32 against symbol `_length_code' can not be used when making a shared object; recompile with -fpic"
